docs/reference/esql/esql-syntax.asciidoc

```diff
@@ -114,6 +114,20 @@ The following boolean operators are supported:
 * `OR`
 * `NOT`
 
+[discrete]
+[[esql-predicates]]
+=== Predicates
+
+For NULL comparison use the `IS NULL` and `IS NOT NULL` predicates:
+
+[source,esql]
+----
+FROM employees
+| WHERE birth_date IS NULL
+| KEEP first_name, last_name
+----
+--
```

Comment:
```suggestion
```

Reply:
This would break the docs. Interesting that CI didn't catch it.

Reply:
That's strange! It should.

I'd move this example into a csv-spec and include it like the function docs do. We probably already have an example in there already.